lib/v2/screens/profile_screens/guardians/guardians_tabs/interactor/viewmodels/page_commands.dart

```diff
@@ -42,3 +42,21 @@ class ShowOnboardingGuardianDoubleAction extends PageCommand {
     required this.rightButtonTitle,
   });
 }
+
+class ShowActivateGuardianDoubleAction extends PageCommand {
+  final int index;
+  final String image;
+  final String description;
+  final String leftButtonTitle;
+  final String rightButtonTitle;
+  final Iterable<GuardianModel> myGuardians;
+
+  ShowActivateGuardianDoubleAction({
```

Comment:
this command doesnt need to tell the UI to use double action. The UI knows what do to. 
The command tells the UI `ShowActivateGuardian`
Then the UI knows its a dialog and its a double action dialog 

lets talk about commands naming during the meeting

Reply:
Fix, change to ShowActivateGuardian.